### PR TITLE
Feature - Issue 181 - Changes Door43 to DCS and adds images

### PR DESCRIPTION
**Pull request by richmahn**
Changes Door43 to "Door43 Content Service" and/or DCS and adds image. Can view at http://test.door43.org:3000 (be sure to logout, see home screen, log in, etc.)

### CODE REVIEW

**Comment by richmahn** [comment]
I just have what is in the Issue

File: public/swagger.v1.json [hunk 1]
```diff
@@ -969,15 +969,12 @@
           "x-go-name": "OpenIssues"
         },
         "owner": {
-          "x-go-name": "Owner",
           "$ref": "#/definitions/User"
```
Comment (phillip-hopper) [inline public/swagger.v1.json]: I don't know the implications of the changes to this file, so I'll defer to @ethantkoenig here.

Comment (ethantkoenig) [inline public/swagger.v1.json]: @richmahn Why did you change this file? It's my understanding that file is automatically generated

Comment (richmahn) [inline public/swagger.v1.json]: Ack. That keeps getting put in. Why is it in the repo if automatically generated? Each time I have to remember to revert it. Forgot this time. Such a pain.

**Comment by richmahn** [comment]
The swagger dagger has been removed.
